src/features/island/hud/components/buildings/Buildings.tsx

```diff
@@ -114,6 +115,19 @@ export const Buildings: React.FC<Props> = ({ onClose }) => {
     );
   };
 
+  const FILTERED_BUILDINGS = () => {
```

Comment:
Similar (same?) function in ListView.tsx -- find a common place to implement once?